openingd/openingd.c

```diff
@@ -113,6 +113,15 @@ struct state {
 	u32 feerate_per_kw_funding;
 };
 
+static struct amount_sat total_funding(struct state *state)
+{
+	struct amount_sat total;
+	assert(amount_sat_add(&total, state->opener_funding,
+			      state->accepter_funding));
```

Comment:
it's a bad habit to wrap active code in assert(), because if you define NDEBUG, assert() vanishes along with its contents.

prefer a simple if(!xxx) abort(); or status_failed().